README.md

```diff
@@ -30,9 +30,9 @@ and visit the page at `localhost:8080` (we recommend in Chrome).
 
 * Set up auto-lint: `echo ./pre-push-hook.sh > .git/hooks/pre-push`
 
-* *[Optional]* Run `./lint.sh --fix` to run clang-format and eslint on all relevant files and format in
-place. Clang-format will run automatically on commit and eslint will run automatically on push so
-this just saves you the extra commits. 
+* *[Optional]* Run `./lint.sh --fix` to run clang-format and eslint on all relevant files and (with `--fix`) format in
+place. Clang-format and eslint will run automatically on `git push` (without `--fix`) so this just
```

Comment:
maybe "Clang-format checking", since clang-format doesn't format in place on pushes